.github/workflows/lint_and_test.yml

```diff
@@ -118,3 +119,55 @@ jobs:
         run: |
           pipenv run integrationtest
 
+  # py_validate:
+  #   name: Validate python
+  #   uses: Energinet-DataHub/.github/.github/workflows/python-pipenv-validate.yaml@main
+  #   with:
+  #     python_version: 3.8
+  #     ignore_lint_error: true
+
+  publish:
+    name: publish
+    runs-on: ubuntu-latest
+    needs: [build, lint, unittest, integrationtest]
+    # needs: [py_validate]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ env.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.python-version }}
+
+      - name: Install pipenv
+        run: |
+          python -m pip install --upgrade pipenv wheel
+
+      - id: cache-pipenv
+        uses: actions/cache@v1
+        with:
+          path: ~/.local/share/virtualenvs
+          key: ${{ runner.os }}-pipenv-${{ hashFiles('**/Pipfile.lock') }}
+
+      - name: Install dependencies
+        run: |
+          pipenv install --deploy --dev
+          pip install pyyaml
+
+      - name: Build prod tarball
+        if: github.event_name == 'push' && github.ref_name == 'main'
+        run: python setup.py sdist
+
+      - name: Build dev tarball
+        if: github.event_name == 'pull_request' && startsWith(github.event.pull_request.head.ref, 'feature/') && github.base_ref == 'main'
+        run: python setup.py sdist --build=.dev${{ github.run_number }}
+
+
+      - name: Publish distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@v1.4.2
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_SECRET }}
+          # TODO: For testing
+          # repository_url: https://test.pypi.org/legacy/
```

Comment:
If you want to keep this around to indicate to others how they can test this. I would say the `TODO` has to go and a better description is needed.

Reply:
fixed